lighthouse-core/audits/accessibility/html-xml-lang-mismatch.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Ensure that HTML elements with both `lang` and `xml:lang`
+ * attributes agree on the base language of the page.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+const AxeAudit = require('./axe-audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that checks if the `xml:lang` language attribute (if it is present) matches the value of the `lang` attribute. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: 'The `<html>` `[lang]` attribute matches the `[xml:lang]` attribute, if present',
```

Comment:
we'll file it under not applicable if it doesn't apply

```suggestion
  title: 'The `<html>` `[lang]` attribute matches the `[xml:lang]` attribute',
```